.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: false
+sudo: true
```

Comment:
Drop this line entirely instead of changing it. It was deprecated/removed from travis, and should have been dropped earlier.